src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -13,104 +13,160 @@
+     */
+    @FXML
+    private URL location;
+
+    /**
+     * Initializes the controller class.
+     * <p>
+     * <a href="https://docs.oracle.com/javase/8/javafx/api/javafx/fxml/Initializable.html">JavaFX Initialize</a>
+     * NOTE This interface has been superseded by automatic injection of location and resources properties into the controller.
+     * FXMLLoader will now automatically call any suitably annotated no-arg initialize() method defined by the controller.
+     * It is recommended that the injection approach be used whenever possible.
+     */
     @FXML
-    public JFXButton resetFilterBtn;
+    private ResourceBundle resources;
 
     @FXML
-    public JFXButton searchBtn;
+    private TextField searchField;
 
     @FXML
-    public TextField searchField;
+    private JFXButton searchBtn;
 
     @FXML
-    public JFXListView<Movie> movieListView;
+    private JFXButton resetFilterBtn;
 
     @FXML
-    public JFXComboBox<Genre> genreComboBox;
+    private JFXButton sortBtn;
+
+    private static final String SORT_DEFAULT_TEXT_ASC = "Sort (asc)";
+    private static final String SORT_DEFAULT_TEXT_DESC = "Sort (desc)";
 
     @FXML
-    public JFXComboBox<Integer> releaseYearPicker;
+    private JFXComboBox<Genre> genreComboBox;
 
     @FXML
-    public JFXComboBox<RatingOption> ratingComboBox;
+    private JFXComboBox<Year> releaseYearPicker;
 
     @FXML
-    public JFXButton sortBtn;
+    private JFXComboBox<Rating> ratingComboBox;
 
-    public List<Movie> allMovies = Movie.initializeMovies();
+    @FXML
+    private JFXListView<Movie> movieListView;
 
     private final ObservableList<Movie> observableMovies = FXCollections.observableArrayList();
-    private final FilteredList<Movie> filteredList = new FilteredList<>(observableMovies);
 
-    @Override
-    public void initialize(URL url, ResourceBundle resourceBundle) {
+    private final FilteredList<Movie> filteredMovies = new FilteredList<>(observableMovies);
+
+    /**
+     * Initializes the controller class.
+     * <p>
+     * <a href="https://docs.oracle.com/javase/8/javafx/api/javafx/fxml/Initializable.html">JavaFX Initialize</a>
+     * NOTE This interface has been superseded by automatic injection of location and resources properties into the controller.
+     * FXMLLoader will now automatically call any suitably annotated no-arg initialize() method defined by the controller.
+     * It is recommended that the injection approach be used whenever possible.
+     */
+    public void initialize() {
+        List<Movie> allMovies;
+
+        try {
+            allMovies = MovieAPIService.getMovies();
+        } catch (IOException e){
+            allMovies = new ArrayList<>();
+        }
+
         observableMovies.addAll(allMovies);
+        sortMovies();
 
-        movieListView.setItems(filteredList);
-        movieListView.setCellFactory(movieListView -> new MovieCell());
+        movieListView.setItems(filteredMovies);
```

Comment:
this is the only place where `filteredMovies` is used, I think you can replace this with `observableMovies` and the functionality will be the same, since we are not applying predicates to the `filteredMovies` list

Reply:
I deleted filtered movies as suggested in WhatsApp